IO-pc-local/src/app/lobby/lobby.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { Student } from './shared/model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LobbyService {
+  private readonly _students: BehaviorSubject<Student[]> = new BehaviorSubject([] as Student[]);
+  public students$: Observable<Student[]> = this._students.asObservable();
```

Comment:
Tutaj tak naprawdę można usunąć explicite typowanie, bo Typescript sam sobie zinterferuje do dokładnie tego samego. Jeśli natomiast wolisz mieć przy każdej zmiennej wprost zdefiniowane, to nie nalegam, bo jest to kwestia gustu

![image](https://github.com/Nepommuck/Software-Engineering-Frontend/assets/100998310/1e2a0466-1310-450e-92e1-98fbcd03cfd5)